src/share/util/shr_scam_mod.F90

```diff
@@ -763,7 +781,8 @@ logical function is_latlon( var_name, latitude, varnotdim )
            trim(var_name) == trim(xyvar)    .or. trim(var_name) == 'lsmlat'    .or. &
            trim(var_name) == trim(gcvar)    .or.                                    &
            trim(var_name) == 'LAT'          .or. trim(var_name) == 'LATIXY'    .or. &
-           trim(var_name) == trim(Capxyvar) .or. trim(var_name) == 'LSMLAT' )  then
+           trim(var_name) == trim(Capxyvar) .or. trim(var_name) == 'LSMLAT'    .or. &
+           trim(var_name) == 'ncol')  then
```

Comment:
is this correct for lat/lon grids? ncol is typically not lat/lon

Reply:
it appears this call is checking to see if the variable is a lat/lon variable in the file and checking for appropriate dimensions for the lat/lon array.  Also, yes, I verified that all A tests in the acme_developer package were b4b with the latest code push.